bitacross-worker/enclave-runtime/src/test/tests_main.rs

```diff
@@ -159,7 +159,7 @@ pub extern "C" fn test_main_entrance() -> size_t {
 		itc_parentchain::light_client::io::sgx_tests::sealing_creates_backup,
 
 		// test musig ceremony
-		bc_musig2_ceremony::sgx_tests::test_full_flow_with_3_ceremonies,
+		// bc_musig2_ceremony::sgx_tests::test_full_flow_with_3_ceremonies,
```

Comment:
This is very important test, we must keep it.

Reply:
I removed some test cause processing logic changed, I will try restore this test

Reply:
Done